src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,88 @@
+    job: JobData
+}
+
+const JobCard: FC<JobCardProps> = ({ job }) => {
+    const handleReadMoreClick = () => {
+        window.open(job.jobPostingUrl, '_blank', 'noopener,noreferrer')
+    }
+
+    return <Card sx={{ bgcolor: 'primary.dark', width: '100%' }}>
+        <CardContent>
+            <Grid item xs={12} md={2} display={{ xs: 'flex', sm: 'flex', md: 'none' }} alignItems="center" justifyContent="right"  sx={{ marginBottom: { xs: 4, sm: 4 }, marginRight: 2 }}>
+                <AccessTimeIcon sx={{ marginRight: 1 }} />
+                <Typography fontSize="body2">
+                    {job.publicationDate}
+                </Typography>
+            </Grid>
+            <Grid container alignItems="center">
+                <Grid item xs={12} sm={12} md={2} display="flex" justifyContent="center">
+                    <Avatar
+                        sx={{ bgcolor: 'white', width: 135, height: 135, borderRadius: 2,  img: {objectFit: 'contain', width: '90%', height: '100%'}
+                        }}
+                        alt={`${job.company} logo`}
+                        src={job.logoUrl}
+                    />
+                </Grid>
+                <Grid item xs={12} sm={12} md={8} mt={2}>
+                    <Typography variant="h5" mb={2} mt={2} component="div" sx={{ textAlign: 'left' }}>
+                        {job.title}
+                    </Typography>
+                    <Grid container spacing={2}>
+                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                            <BusinessIcon />
+                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.company}</Typography>
+                        </Grid>
+                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                            <LocationOnIcon />
+                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.location}</Typography>
+                        </Grid>
+                        {job.salary != '' &&
+                            <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                <AccountBalanceWalletIcon />
+                                <Typography fontSize="body2" sx={{ ml: 1 }}>{job.salary}</Typography>
+                            </Grid>
+                        }
+                    </Grid>
+                    <Grid direction='row' spacing={1} mt={2} justifyContent="flex-start" flexWrap="wrap">
+                        {job.tags.map((tag, index) => (
+                            <Chip key={index} label={tag} sx={{ bgcolor: 'secondary.dark', height: 32, borderRadius: '8px', mt: { xs: 2, sm: 0 }, ml: { xs: 1, sm: 1 }, flexShrink: 0 }} />
+                        ))}
+                    </Grid>
```

Comment:
Let's use the `Stack` here, as Stacks are 1 dimensional and Grids are 2D. Then we can remove the left margin that is bumping the tags out of alignment with our grid (imagine an invisible vertical rule that connects the title, the icons, and the tags). The Stack automatically adds the in-between spacing for us.

If you'd like to add a top margin let's put it on the grid rather than adding it multiple times on each tag.

<img width="1134" alt="Screenshot 2024-09-01 at 1 43 44 PM" src="https://github.com/user-attachments/assets/259f75d0-0621-431f-b4b4-7d914594f37e">


```suggestion
                    <Stack direction='row' spacing={1} mt={2} justifyContent="flex-start" flexWrap="wrap">
                        {job.tags.map((tag, index) => (
                            <Chip key={index} label={tag} sx={{ bgcolor: 'secondary.dark', height: 32, borderRadius: '8px', flexShrink: 0 }} />
                        ))}
                    </Stack>
```